signer/apk/jar.go

```diff
@@ -22,8 +22,12 @@ func makeJARManifests(input []byte) (manifest, sigfile []byte, err error) {
 	mw := bytes.NewBuffer(manifest)
 
 	for _, f := range r.File {
-		// reserved signature files do not get included in the manifest
 		if isSignatureFile(f.Name) {
+			// reserved signature files do not get included in the manifest
+			continue
+		}
+		if f.FileInfo().IsDir() {
+			// directories do not get included
```

Comment:
Are there any other considerations required here for symbolic links or other file types that are not regular files?

Reply:
I don't think so. Java does not honor symlinks in JAR files, so I wouldn't worry about them too much.